content/user/apps/Route-coverage.md

```diff
@@ -0,0 +1,40 @@
+<!--
+title: "Route Coverage"
+description: "Overview of setting up environments"
+tags: "user UI applications route coverage exercised vulnerabilities"
+-->
+
+## About Route Coverage
+
+An application “route” is a combination of three distinct data points: the normalized URL of the route, the HTTP verb associated with the request (e.g., Get or Post) and the method signature of the controller action. When you consistently exercising each route in your application, the Contrast agent can successfully Assess and Protect the surface layer of your application, and discover vulnerabilities. 
+
+### Agent configuration 
+
+Contrast supports route coverage for the following frameworks: 
+
+* Java: Spring Spring MVC 4.x 
+* .NET: ASP.NET MVC (versions 4 and 5), WebForms, WebAPI and WCF
+* Node: Express 
+* Ruby: Rails and Sinatra
+* Python: Django, Pyramid and Flask
+
+Coverage is enabled automatically for .NET or Node applications. For Java applications, you must use the following command to specify the application name when deploying the Java agent: `-Dcontrast.appname = <example name>`. 
```

Comment:
Coverage is automatic for Ruby & Python as well. Do we only want to mention the Java part since it deviates from the norm?